packages/react-components/react-radio/src/components/RadioGroup/RadioGroup.test.tsx

```diff
@@ -87,6 +87,20 @@ describe('RadioGroup', () => {
     expect(items[2].required).toBe(true);
   });
 
+  it('applies aria-describedby to every radio item', () => {
+    const { getAllByRole } = render(
+      <RadioGroup aria-describedby="test-describedby">
+        <Radio />
+        <Radio />
+        <Radio />
+      </RadioGroup>,
+    );
+    const items = getAllByRole('radio') as HTMLInputElement[];
+    expect(items[0].getAttribute('aria-describedby')).toEqual('test-describedby');
+    expect(items[1].getAttribute('aria-describedby')).toEqual('test-describedby');
+    expect(items[2].getAttribute('aria-describedby')).toEqual('test-describedby');
+  });
+
```

Comment:
If you go with my comment above, you should also add a test that sets `aria-describedby` individually to a `Radio` component aside from the one coming from the `RadioGroup` and ensure they are merged correctly.

Reply:
Updated to test that aria-describedby can be overridden by an individual radio item.